public/main.js

```diff
@@ -0,0 +1,54 @@
+// setting the map's view to Burlington
+let myMap = L.map("map").setView([44.4798, -73.2143], 14)
+
+// adding tile layet
+L.tileLayer('https://{s}.tile.opentopomap.org/{z}/{x}/{y}.png', {
+  maxZoom: 17,
+  attribution: 'Map data: &copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, <a href="http://viewfinderpanoramas.org">SRTM</a> | Map style: &copy; <a href="https://opentopomap.org">OpenTopoMap</a> (<a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-SA</a>)'
+}).addTo(myMap)
+
+// adding marker to downtown Burlington on map
+let marker = L.marker([44.4798, -73.2143])
+let downtown = "Downtown Burlington"
+marker.bindPopup(`${downtown}`)
+marker.addTo(myMap)
```

Comment:
With how many markers you drop with all of the restaurants, having an extra one just to show where downtown is adds clutter but there's also not a major downside to having this here either